rfcs/tag-component.md

```diff
@@ -0,0 +1,61 @@
+# Tag Component
+
+### Exports
+
+- `Tag`
+- `StyledRoot`
+- `StyledCross`
+
+### `Tag` API
+
+- `color: STYLE.blue(theme.colors.primary470) | STYLE.orange(theme.colors.warning400) | STYLE.green(theme.colors.positive400) | STYLE.red(theme.colors.negative400)`:
+  the color theme to be applied to a Tag. Default is `STYLE.blue`.
+- `children: React$Node`:
+  Component or String value for label of tag. Default is empty string
+- `disabled: boolean`:
+  Disable control from being changed
+- `overrides: {Root: (props: {[string]: any}) => React$Node, Cross: (props: {[string]: any}) => React$Node}`
+  - Custom `Cross` button.
+  - `Root` wrapper element for the whole tag control to apply styles
+- `onCrossClick: func(remove => {})`:
+  handler for events on Cross button element. `remove` argument method can be called any time to remove tag.
+- `onMouseEnter: func`:
+  handler for events on trigger element
+- `onMouseLeave: func`:
+  handler for events on trigger element
+- `onFocus: func`:
+  handler for events on trigger element
+- `onBlur: func`:
+  handler for events on trigger element
```

Comment:
Is it worth making these a part of the explicit API? Technically the user can just do overrides for whatever event handler they want:

```js
<Tag overrides={{Root: {
  props: {onMouseEnter: myHandler}
}}}>
  Foo
</Tag>
```

Reply:
+1 I think this isn't necessary and could just be pass-thru via props override

Reply:
ok